nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-jetty/src/test/groovy/org/apache/nifi/web/server/JettyServerGroovyTest.groovy

```diff
@@ -540,6 +540,21 @@ class JettyServerGroovyTest extends GroovyTestCase {
         assert filterNames.contains("DoSFilter")
         assert !filterNames.contains("ContentLengthFilter")
     }
+
+    @Test
+    void testDetermineRequestTimeoutInMilliseconds() {
+        // Arrange
+        Map badProps = [
+                (NiFiProperties.WEB_REQUEST_TIMEOUT): "50 sec",
+        ]
+
+        NiFiProperties mockProps = new StandardNiFiProperties(new Properties(badProps))
+
+        // Act
+        long requestTimeout = JettyServer.determineRequestTimeoutInMilliseconds(mockProps)
```

Comment:
Recommend avoiding calls to private methods even though Groovy allows it to work.  Instead, is it possible to retrieve the configured DoSFilter and evaluate the configuration?  Otherwise, it seems acceptable to remove this test method.

Reply:
Sure I can remove, used this initially to check the value was coming through correctly.